product_docs/docs/biganimal/release/administering_cluster/projects.mdx

```diff
@@ -48,9 +48,11 @@ Before creating a project:
 
 To create a new project:
 
-1. Select **Projects > New Project**.
-1. Enter a unique name for the project.
-1. Select **Create Project**.
+1. Select **Projects > Create New Project**.
+1. In the **Project Name** field, enter a unique name.
+1. Under **Tags*, select **+**.
```

Comment:
attaching tags is not a required step when create cluster, can we make it optional?